previousnext/agov
# 32436 news media page frontend fixes

Steps to test:

1)  Go to /news-media
   a) check page has an intro para
   b) check that the icon boxes do not have a border nor bg on hover

![](https://dl.dropbox.com/s/ydgff69yuy76l5k/Screenshot%202015-07-22%2014.54.07.png?dl=0)

2) Check that promo regions have been added and promo views has been removed from /front and /news-media
     ![](https://dl.dropbox.com/s/gbopxxr2dgfhvqb/Screenshot%202015-07-22%2015.00.35.png?dl=0)

3) Check that box icons on the front page are themed as previously - with border and bg on hover.
    ![](https://dl.dropbox.com/s/sgn4l6lz0fhrzh5/Screenshot%202015-07-22%2015.02.41.png?dl=0)

4) Check that promo beans on the front page and on /news-media are themed as below and title + image link to its reference entity

![](https://dl.dropbox.com/s/10se09z90p5kd0c/Screenshot%202015-07-22%2015.04.22.png?dl=0)

6) Check there are borders news-media page as follows:
    ![](https://dl.dropbox.com/s/2ezcnb6g660g911/Screenshot%202015-07-23%2016.04.29.png?dl=0)
